src/module.pegjs

```diff
@@ -528,16 +528,26 @@ zone_connect_pads
     return { type, value }
 }
 
+generator
+    = "(" _
+        type:"generator" _
+        value:symbol _
+        ")" {
+            return { type, value }
+        }
 
 header /* parseHeader */
-  =  version:(header_version _ )?
-        "(" _ symbol _ (string/symbol) _ (string/symbol _)? ")"
+  =  "(" _ generator:symbol _ layer:(string/symbol) _ attr:(string/symbol _)? ")"
     {
-    var out = []
-    if(version)
-        out.push(version[0])
-    return out
-};
+        return {
+            type: "header",
+            value: {
+                generator: { type: "string", value: generator },
+                layer: { type: "string", value: layer },
+                attr: { type: "string", value: attr }
+            }
+        };
+    };
```

Comment:
So the header version won't be treated specially anymore and is just a good old fashion value from the standpoint of this new parser?

Reply:
Yea... version was actually defined in the header, as well as  in the module contents below... see line 576...